spec/features/ingredients/edit_ingredient_page_spec.rb

```diff
@@ -0,0 +1,34 @@
+feature 'user visits edit ingredients page' do
+  before do
+    @ingredient = FactoryBot.create(:ingredient, :egg)
```

Comment:
If you want to have variable accessible outside of the block in your `it` blocks then use `let`

```ruby

let!(:ingredient) { FactoryBot.create(:ingredient, :egg) }

it 'aaa' do
  # use ingredient here.
end
```